packages/pwa/package.json

```diff
@@ -188,26 +188,16 @@
         ],
         "defaultCurrency": "GBP",
         "supportedLocales": [
-            {
-                "id": "en-GB",
-                "preferredCurrency": "GBP"
-            },
-            {
-                "id": "fr-FR",
-                "preferredCurrency": "EUR"
-            },
-            {
-                "id": "it-IT",
-                "preferredCurrency": "EUR"
-            },
-            {
-                "id": "zh-CN",
-                "preferredCurrency": "CNY"
-            },
-            {
-                "id": "ja-JP",
-                "preferredCurrency": "JPY"
-            }
+            { "id": "de-DE", "preferredCurrency": "EUR" },
```

Comment:
Looks like code format is different with the changes, it was supposed to create new line on the '{'. Maybe you want to check your prettier?